taas/config.d/organizations.yml

```diff
@@ -4,30 +4,49 @@ sources:
         options:
             build_by_default: False
         beta-v1:
-            url: https://docs.google.com/spreadsheets/d/1oa7Wknz64dMQlV2XsW0ssjh6Gw5SnHVfGxDSw_5eKOQ/edit#gid=518938746
+            url: https://docs.google.com/spreadsheets/d/1GIA3FrRM1GZS3FoUDNpw35MhC_WLTdKBiiU2tUlWS84/edit#gid=0
             mapping:
-                id: HR info ID
+                id: ID
+                hrinfo_id:
+                    field: HRinfo ID
+                    optional: True
+                rw_id:
+                    field: RW ID
+                    optional: True
+                ocha_gms_id:
+                    field: OCHA_GMS ID
+                    optional: True
                 label: Preferred Term
+                alt_label.default:
+                    field: Alt Term
+                    optional: True
+                alt_label.hrinfo:
+                    field: HRinfo Alt Term
+                    optional: True
+                alt_label.gms:
+                    field: GMS Alt Term
+                    optional: True
+                alt_label.rw:
+                    field: RW Alt Term
+                    optional: True
+                alt_label.fts:
+                    field: FTS Alt Term
+                    optional: True
+                url:
+                    field: URL
+                    optional: True
                 acronym:
-                    type: map
-                    field: ACRONYM
-                    optional: True
-                homepage:
-                    type: map
-                    field: homepage
-                    optional: True
-                fts_id:
-                    type: map
-                    field: FinancialTrackingService ID
-                    optional: True
-
-                # This gives us a map with a machine-friendly URL,
-                # and a human-friendly ID.
-                type.id:
-                    type: link
-                    field: OrgTypeID - OrgTypePreferredTerm
-                    prefix: https://www.humanitarianresponse.info/en/api/v1.0/organization_types/
-                type.label:
-                    type: link
-                    field: OrgTypeID - OrgTypePreferredTerm
-                    from: label
+                    field: Acronymn
+                    optional: True
+                type.default:
+                    field: Organization Type
+                    optional: True
+                type.rw:
+                    field: RW Type
+                    optional: True
+                based_in:
```

Comment:
`based_in` looks like it's a country link! We can make it clear that this refers to a country using the special link type, complete with a URL!

```YAML
based_in.country.id:
    type: link
    field: Based in
    optional: True
    prefix: http://vocabulary.unocha.org/json/beta-v1/countries/

based_in.country.label:
    type: link
    field: Based in
    optional: True
    from: label
```

Having `http://vocabulary.unocha.org/json/beta-v1/countries/` work for country fragments is on my list of things to do today.

The link also means we know this is a *country* rather than an *operation*, which often look the same. (I'm assuming it's a country based upon the country having a countries validation sheet that feeds this field.)